### DISCUSSION TITLE
MMPretrain CUDA available: False

### DISCUSSION THREAD

**Discussion by L-A-Sandhu** [open-mmlab/mmclassification#1579] [General]
I am writing a code for image classification using a swin transformer. However, it says CUDA available: False. whereas, i have 3060 installed in the system that works well with TensorFlow and Pytorch frameworks. 
Please check if my GPU is in use as my training crashes after first epoch 
![image](https://github.com/open-mmlab/mmpretrain/assets/94599611/bce96f38-33e2-4d5e-9f8a-5a9ea0254dbb)

**Comment by Ezra-Yu** [comment]
try:

```
>>> import torch
>>> torch.cuda.is_available()
True
```

If the output is False, there maybe some thing wrong with your installation with your torch. please refer to the PyTorch for more help.
